chapters/VK_EXT_acquire_wl_display/acquire_wl_display.txt

```diff
@@ -0,0 +1,66 @@
+[open,refpage='vkAcquireWaylandDisplayEXT',desc='Acquire access to a VkDisplayKHR using Wayland',type='protos']
+--
+
+To acquire permission to directly access a display in Vulkan from a Wayland
+compositor, call:
+
+include::{generated}/api/protos/vkAcquireWaylandDisplayEXT.txt[]
+
+  * pname:physicalDevice The physical device the display is on.
+  * pname:display A wl_display connected to the Wayland compositor that
+	currently owns pname:display.
+  * pname:manager A zwp_drm_lease_manager_v1 connected to the Wayland compositor
+	that owns the connectors requested for lease.
```

Comment:
If I remember correctly, `zwp_` indicates an experimental Wayland extension, correct?  If so, that's kind-of a problem.  We don't really want to bake something into a Vulkan API extension and ship it in drivers when the Wayland extension is liable to change out from under us.

Reply:
I addressed this in the OP:

>I want to avoid creating a Vk extension which depends on an unstable Wayland protocol, as the structure names reflect their unstable nature and I would prefer not to immortalize the unstable version in this extension. Therefore, once consensus is reached on the Wayland protocol, we're thinking about fast-tracking it to stable. At that point, I'll update this patch accordingly.

Reply:
Oh, I see your commentary above about experimental stuff.  Sorry for not reading :-/  As long as you've reserved an extension number, stuff can sit in branches indefinitely until you're ready to call it all baked.

Reply:
Cool, cheers!